src/styles/_previwer.scss

```diff
@@ -0,0 +1,27 @@
+.previewer {
```

Comment:
the file name should be `_previewer.scss`

Reply:
Ohh 😅

You are right!